acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
No, we specifically do _not_ want this to go into beaker itself.  The principle @joshcooper has proposed that I think we should be following is that beaker does not have any built-in knowledge about the layout of puppetserver or puppet-agent.

That is to say, beaker should inspect path values dynamically using mechanisms like `puppet config print`.

We don't have anything like `puppet config print` for puppetserver printing out hte value of our configuration directory, so we're providing that information as something like a command line argument, but not a command line argument because a true argument would require changes to the jenkins jobs.

The options.rb file is the best place we have today to make this sort of change and manage it going forward.